sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 12.7.0-beta.1 (Unreleased)
-
+- Fixed a bug that would cause auth failures when building a client by passing an endpoing which had a sas token with protocol set to https,http
```

Comment:
```suggestion
- Fixed a bug that would cause auth failures when building a client by passing an endpoint which had a sas token with protocol set to https,http
```